479 add `renv::snapshot()` for `.lockfile` with `future` + `shiny::ExtendedTask` (#1232)
insightsengineering/teal: NEWS.md

# teal 0.15.2.9039

### New features

> * Footer of `teal` apps is enriched with a modal that shows `.lock` file created with `pak` package. 

Reviewer: this seems old. this feature now using `renv `right? Please update it.

Author: oh yes!

Author: 32dc638fcad93c316093043d5619009274992c37

### Enhancement
* Provided progress bar for modules loading and data filtering during teal app startup.

### New features

> * Footer of `teal` apps is enriched with a button that allows to download the `.lock` file created with `renv` package. 
> This will boost the reproducibility of the app.

Reviewer 2: Make news entry as concise as possible and try to avoid stuff that does not bring that much or tries to value changes made.

```suggestion
* Possibility to download `.lock` file to restore app session for reproducibility.
```
